docs/grid_model/network_subnetwork.md

```diff
@@ -975,6 +976,17 @@ between the converter DC Node 1 and the DC Node 2 to be equal to `TargetVdc`
   - `+TargetVdc / 2` at the converter DC Node 1
   - `-TargetVdc / 2` at the converter DC Node 2
 
+When the `ControlMode` of the converter is set to `P_PCC_DROOP`, the converter controls active power as in the `P_PCC` control mode
+for normal load flow, but when a security analysis in run, the converter controls the relation between DC Voltage and DC Power:
+$P_{DC} - P_{REF} = -k * (V_{DC} - V_{REF})$
+Where $k$ is the droop coefficient of the actual droop segment.
```

Comment:
I am not sure to fully understand:
- can you define more precisely what are Pdc, Pref, Vdc, Vref ?
- do some of them relate to or equal to targetP / targetVdc ?
- you mention distinction between load flow and security analysis, are some of these values the solved values from the base case load flow? (e.g. Pref is solved Pdc after load flow ?)